pkg/executor/executortype/newdeploy/newdeploymgr.go

```diff
@@ -573,7 +574,8 @@ func (deploy *NewDeploy) updateFunction(oldFn *fv1.Function, newFn *fv1.Function
 
 	deployChanged := false
 
-	if oldFn.Spec.InvokeStrategy != newFn.Spec.InvokeStrategy {
+	// TODO : Replace with custom equal function if required
```

Comment:
I do think custom function would be better because reflection doesn't show errors during compilation and is a performance intensive package. Please let me know if this approach is worth it